ci-operator/config/openshift/cluster-cloud-controller-manager-operator/openshift-cluster-cloud-controller-manager-operator-master.yaml

```
@@ -68,16 +68,16 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-ccm
-- as: e2e-openstack
-  steps:
-    cluster_profile: openstack-vexxhost
-    env:
-      TEST_SUITE: experimental/reliability/minimal
-    workflow: openshift-e2e-openstack-ipi
 - as: e2e-aws-ccm-install
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-ccm-install
+- as: e2e-openstack-ccm
+  steps:
+    cluster_profile: openstack-vexxhost
```

```suggestion
    cluster_profile: openstack
```

Let's try it for now

---

I'm trying vh-mecha. MOC is sadness. I don't want to run anything there.